DSA/1.1 Arrays/3. Range Queries/DSA_1.1.3.4_Arrays_ Range_Queries_#410.md

```diff
@@ -0,0 +1,184 @@
+## Problem Statement
+Given an array of N elements and you have to answer Q queries of the form L R K , To Count the numbers 
```

Comment:
update the problem statement something like explanation of merge sort tree and it's implementation , rest is looking fine.